### ISSUE TITLE
ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found

### ISSUE THREAD

**Issue by kvnptl** [b-it-bots/docker#26]
While running YoloV5 code on at-work lab GPU workstation, the jupyter notebook is giving the following error:

```
Traceback (most recent call last):
  File "/home/jovyan/public/b_it_bot_work/2d_perception/yolov5/train.py", line 40, in <module>
    import val  # for end-of-epoch mAP
  File "/home/jovyan/public/b_it_bot_work/2d_perception/yolov5/val.py", line 37, in <module>
    from models.common import DetectMultiBackend
  File "/home/jovyan/public/b_it_bot_work/2d_perception/yolov5/models/common.py", line 27, in <module>
    from utils.plots import Annotator, colors, save_one_box
  File "/home/jovyan/public/b_it_bot_work/2d_perception/yolov5/utils/plots.py", line 17, in <module>
    import seaborn as sn
  File "/opt/conda/lib/python3.9/site-packages/seaborn/__init__.py", line 2, in <module>
    from .rcmod import *  # noqa: F401,F403
  File "/opt/conda/lib/python3.9/site-packages/seaborn/rcmod.py", line 7, in <module>
    from . import palettes
  File "/opt/conda/lib/python3.9/site-packages/seaborn/palettes.py", line 9, in <module>
    from .utils import desaturate, get_color_cycle
  File "/opt/conda/lib/python3.9/site-packages/seaborn/utils.py", line 10, in <module>
    from scipy import stats
  File "/opt/conda/lib/python3.9/site-packages/scipy/__init__.py", line 211, in __getattr__
    return _importlib.import_module(f'scipy.{name}')
  File "/opt/conda/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/opt/conda/lib/python3.9/site-packages/scipy/stats/__init__.py", line 467, in <module>
    from ._stats_py import *
  File "/opt/conda/lib/python3.9/site-packages/scipy/stats/_stats_py.py", line 46, in <module>
    from . import distributions
  File "/opt/conda/lib/python3.9/site-packages/scipy/stats/distributions.py", line 8, in <module>
    from ._distn_infrastructure import (rv_discrete, rv_continuous, rv_frozen)
  File "/opt/conda/lib/python3.9/site-packages/scipy/stats/_distn_infrastructure.py", line 24, in <module>
    from scipy import optimize
  File "/opt/conda/lib/python3.9/site-packages/scipy/__init__.py", line 211, in __getattr__
    return _importlib.import_module(f'scipy.{name}')
  File "/opt/conda/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/opt/conda/lib/python3.9/site-packages/scipy/optimize/__init__.py", line 413, in <module>
    from ._linprog import linprog, linprog_verbose_callback
  File "/opt/conda/lib/python3.9/site-packages/scipy/optimize/_linprog.py", line 21, in <module>
    from ._linprog_highs import _linprog_highs
  File "/opt/conda/lib/python3.9/site-packages/scipy/optimize/_linprog_highs.py", line 20, in <module>
    from ._highs._highs_wrapper import _highs_wrapper
ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /opt/conda/lib/python3.9/site-packages/scipy/optimize/_highs/_highs_wrapper.cpython-39-x86_64-linux-gnu.so)
```

## Describe the bug

- Upon checking the existing version of GLIBCXX,
Command: `!strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX`
Output:
```
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.23
GLIBCXX_3.4.24
GLIBCXX_3.4.25
GLIBCXX_3.4.26
GLIBCXX_3.4.27
GLIBCXX_3.4.28
GLIBCXX_DEBUG_MESSAGE_LENGTH
```

## Steps to Reproduce

1. Jupyter notebook link: `/home/jovyan/public/b_it_bot_work/2d_perception/custom_object_detection_using_YOLOv5_opencv.ipynb`

## Environment

* Version used (branch/commit): master
* PC:
    - [x] C025 lab PC

**Comment by kvnptl** [comment]
Possible solution from github: https://github.com/lhelontra/tensorflow-on-arm/issues/13

**Comment by mhwasil** [comment]
Downgrading scipy version to 1.8.1 solves this issue

**Comment by mhwasil** [comment]
This problem is fixed in #27. I am closing this issue
